### ISSUE TITLE
Tweak / plot size in vignette plots

### ISSUE THREAD

**Issue by dylanhmorris** [CDCgov/ww-inference-model#64]
At least on my / @damonbayer's machines, vignette plots are particular readable as they render within the `pkgdown` website:

![unnamed-chunk-13-2](https://github.com/user-attachments/assets/0e9d2801-8688-425f-909c-85bcf7c64ffc)

I have fixed the pixelation issue in #62 but there's also the question of the relative sizes of points, facet labels, axes labels, etc.

I note that plots are set up to be full width:
https://github.com/CDCgov/ww-inference-model/blob/142b5572e66dd09893d646a558b478985baeb900/vignettes/wwinference.Rmd#L157

**Comment by kaitejohnson** [comment]
Yeah I don't know how to fix this! They look horrible -- I should test if they look this bad when I render it to a pdf/html because they look fine when I just run the Rmd and view the plots.

**Comment by dylanhmorris** [comment]
Have you tried previewing the packagedown site itself locally? https://pkgdown.r-lib.org/reference/preview_site.html

**Comment by kaitejohnson** [comment]
Brilliant, didn't know you could do that. Will make a PR to fix this
